classical-components-demo/ocr/src/main/java/io/scanbot/example/MainActivity.kt

```diff
@@ -134,40 +145,45 @@ class MainActivity : AppCompatActivity() {
     This AsyncTask is used here only for the sake of example. Please, try to avoid usage of
     AsyncTasks in your application
      */
-    private inner class RecognizeTextWithPDFTask private constructor(private val imageUri: Uri) : AsyncTask<Void, Void, OcrResult?>() {
-        override fun doInBackground(vararg voids: Void): OcrResult? {
-            return try {
-                val bitmap = loadImage()
+    private suspend fun recognizeTextWithPDFTask(imageUri: Uri) {
```

Comment:
Rename method - it has word 'task' in it after asynctask, which is no more relevant. Rename here and in other modules